glide.yaml

```diff
@@ -113,4 +113,10 @@ import:
   - daemon
 - package: github.com/google/go-github
 - package: github.com/hashicorp/go-version
-- package: github.com/mvdan/xurls
\ No newline at end of file
+- package: github.com/mvdan/xurls
+- package: github.com/prometheus/client_golang
```

Comment:
Do we need the client, if we're going with go-kit? Shouldn't go-kit bring it's dependencies?